Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbon.cs

```diff
@@ -2908,7 +2908,7 @@ private bool TabIsContextValid(KryptonRibbonTab? tab)
                 if (!string.IsNullOrEmpty(SelectedContext))
                 {
                     // Get list of all valid contexts
-                    var contexts = SelectedContext.Split(',');
+                    var contexts = SelectedContext!.Split(',');
 
                     // If the tab context name is one of the selected contexts, then fine
                     foreach (var context in contexts)
```

Comment:
This could mean that `contexts` is null, which will make the foreach throw an exception.